runtime/queries/php/highlights.scm

```diff
@@ -8,6 +8,14 @@
 (named_type (name) @type) @type
 (named_type (qualified_name) @type) @type
 
+(namespace_definition
+  name: (namespace_name(name)) @constructor
+)
```

Comment:
I'm not very familiar with php but wouldn't `@namespace` be a more appropriate capture here?